runtime/polkadot/src/lib.rs

```diff
@@ -1211,15 +1225,22 @@ impl paras_registrar::Config for Runtime {
 	type WeightInfo = weights::runtime_common_paras_registrar::WeightInfo<Runtime>;
 }
 
+pub const LEASE_PERIOD: BlockNumber = 12 * WEEKS;
+
+#[cfg(feature = "fast-runtime")]
```

Comment:
you can, in principle, also make this be generated by your macro to achieve a cleaner code.